# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: layout/forms/nsFieldSetFrame.h
# Commit: 867abe6c80a3
# Full Hash: 867abe6c80a36d66fec92f276c3a7d7c315637bf
# Author: Mats Palmgren <mats@mozilla.com>
# Date: 2021-03-01 09:36:12
# Description:
#   Bug 1694459 - Deal with 'display:contents' style changes on <fieldset> descendants better where it affects which <legend> is the rendered legend.  r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D106566
# ==============================================================================

diff -r 77247863a2ea -r 867abe6c80a3 layout/forms/nsFieldSetFrame.h
--- a/layout/forms/nsFieldSetFrame.h	Mon Mar 01 01:16:36 2021 +0000
+++ b/layout/forms/nsFieldSetFrame.h	Mon Mar 01 03:42:41 2021 +0000
@@ -53,10 +53,10 @@
                                    nsFrameList& aChildList) override;
   virtual void AppendFrames(ChildListID aListID,
                             nsFrameList& aFrameList) override;
-#ifdef DEBUG
   virtual void InsertFrames(ChildListID aListID, nsIFrame* aPrevFrame,
                             const nsLineList::iterator* aPrevFrameLine,
                             nsFrameList& aFrameList) override;
+#ifdef DEBUG
   virtual void RemoveFrame(ChildListID aListID, nsIFrame* aOldFrame) override;
 #endif
 